dipy/align/imwarp.py

```diff
@@ -1264,6 +1255,38 @@ def _iterate(self):
 
         self.energy_list.append(fw_energy + bw_energy)
 
+        self.__invert_models(current_disp_world2grid, current_disp_spacing)
+
+        # Free resources no longer needed to compute the forward and backward
+        # steps
+        if self.callback is not None:
+            self.callback(self, RegistrationStages.ITER_END)
+        self.metric.free_iteration()
+
+        return der
+
+    def __set_no_boundary_displacement(self, step):
+        """ set zero displacements at the boundary
+
+        :param ndarray step:
+        :return ndarray:
```

Comment:
Can you update the docstring by using  [Numpy documentation standard](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt)